Client/src/Components/Task/Task.jsx

```diff
@@ -116,7 +123,7 @@ export default function Task(props){
           />
           <span className="line-divider"></span>
           <span className="elapsed-time">Time: {props.elapsedTime}</span>
-          <img className="options" src="https://app.clockify.me/assets/ui-icons/menu-dots-vertical.svg" alt="" />
```

Comment:
It seems that the three dots image was deleted here?  Was that purposeful?

Reply:
Yes I replaced the image with OptionsMenu component